package.json

```diff
@@ -10,21 +10,19 @@
         "production": "mix --production"
     },
     "devDependencies": {
-        "axios": "^0.21",
+        "axios": "^0.25",
         "laravel-mix": "^6.0.6",
         "lodash": "^4.17.19",
-        "postcss": "^8.1.14"
+        "postcss": "^8.1.14",
+        "resolve-url-loader": "^5.0.0",
+        "sass": "^1.52.2",
+        "sass-loader": "^12.6.0"
     },
-  "name": "ztl-v2",
-  "description": "## vZTL Website",
-  "version": "1.0.0",
-  "main": "webpack.mix.js",
-  "directories": {
+    "name": "ztl-v2",
+    "description": "## vZTL Website",
+    "version": "1.0.0",
+    "main": "webpack.mix.js",
+    "directories": {
     "test": "tests"
-  },
-  "dependencies": {
-    "imagemin-pngquant": "^5.1.0",
```

Comment:
imagemin-pngquant has been a thorn in my side for the last month. My next step on this project was to figure out if this was still required and remove it. Thanks!

Reply:
Yeah everything still appears to work without it. In QA, just make sure it didn't break anything (since you're a little more familiar with where JS is used than I am at the current state), but from my testing it appeared to be fine without it.